testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/HostExcludesTestCase.java

```diff
@@ -187,12 +187,13 @@
                 "org.wildfly.extension.microprofile.reactive-streams-operators-smallrye"
         )),
         WILDFLY_24_0("WildFly24.0", WILDFLY_23_0),
-        // If an extension is added to this enum, also check if it is supplied only by wildfly-galleon-pack. If so, add it also
-        // to the internal mpExtensions Set defined on this class.
-        CURRENT(MAJOR, WILDFLY_24_0, Arrays.asList(
+        WILDFLY_25_0("WildFly25.0", WILDFLY_24_0, Arrays.asList(
                 "org.wildfly.extension.elytron-oidc-client",
                 "org.wildfly.extension.opentelemetry"
-        ));
+        )),
+        // If an extension is added to this enum, also check if it is supplied only by wildfly-galleon-pack. If so, add it also
+        // to the internal mpExtensions Set defined on this class.
+        CURRENT(MAJOR, WILDFLY_25_0);
```

Comment:
this commit can be dropped now; it's in main

Reply:
Updated